.github/workflows/scala.yml

```diff
@@ -215,3 +227,125 @@ jobs:
           text
           EOF
 
+  build-launcher:
+    name: Build Launcher
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        os: [macOS-latest, ubuntu-latest, windows-latest]
+      fail-fast: false
+    steps:
+      - name: Checkout (PR)
+        uses: actions/checkout@v2
+        if: github.event_name == 'pull_request'
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Checkout (Branch)
+        uses: actions/checkout@v2
+        if: github.event_name != 'pull_request'
+      - name: Enable Developer Command Prompt (Windows)
+        uses: ilammy/msvc-dev-cmd@v1.3.0
+      - name: Setup conda
+        uses: s-weigand/setup-conda@v1
+        with:
+          update-conda: true
+          conda-channels: anaconda, conda-forge
+      - name: Setup Conda Environment on Windows
+        if: runner.os == 'Windows'
+        run: |
+          conda create --name enso
+          conda init powershell
+      - name: Activate Conda Environment on Windows
+        if: runner.os == 'Windows'
+        run: conda activate enso
+      - name: Install FlatBuffers Compiler
```

Comment:
You don't really need these

Reply:
Actually I do, because sometime ago we added a 'version check' on sbt load which checks if the external requirements (GraalVM and flatc) are installed and have correct versions. The goal was to be more user friendly for developers, as you are unlikely to want to build the project without some of them. But then to satisfy this version check, I need flatc installed even in this workflow.
An alternative could be to create a dummy `flatc` executable that just prints the correct version to satisfy the version check, or removing the version check, but I think that this is the cleanest solution.

Reply:
Alternatively, I could augment the check to ignore checking flatbuffers if an environment variable is set `IGNORE_FLATC`, what do you think about that?

Reply:
Environment var sounds good. But also, what was the reasoning to check the version before launch and not when it's actually needed?

Reply:
When compiling Enso from scratch the compilation will take some time and the flatbuffers error will be reported not at the very beginning but somewhere halfway through. I think that the idea is to report the missing dependencies as soon as possible to avoid confusion.

Maybe indeed it is just worth removing the check on load? (It is still checked independently when actually running flatc)

Reply:
> to check the version before launch and not when it's actually needed

I also vote for lazy check

Reply:
The on-load flatc check is being removed. It seems that it is completely enough to do the version check before launching flatc and report any errors then, instead of requiring flatc even if some reasonable usecases may not need it.